bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1289,14 +1298,14 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
          ic_ymd="18500101" sim_year="1850" do_transient_pfts=".false." use_excess_ice=".true."
          ic_tod="0" glc_nec="10" use_crop=".true." irrigate=".false."
```

Comment:
Below I suggest that we don't bring in the ctsm5.4 surface datasets. However, I think we *should* bring in the finidat files to get us closer to that work. So these changes would stay.

But, they each should have

`use_init_interp=".true."`

Which it looks like they do, except this first one.

Reply:
Ok, added it.